Regulator_Linear.dcm

```diff
@@ -6972,6 +6972,24 @@ K linear low dropout Regulator fixed
 F http://www.ti.com/lit/ds/symlink/tps722.pdf
 $ENDCMP
 #
+$CMP TPS73018DBV
+D Low Noise, High PSRR, 200mA Low Drop 1.8V Regulator, SOT-23-5
+K Fixed LDO 200mA
+F https://www.ti.com/lit/ds/symlink/tps730.pdf
+$ENDCMP
+#
+$CMP TPS730285DBV
+D Low Noise, High PSRR, 200mA Low Drop 2.85V Regulator, SOT-23-5
+K Fixed LDO 200mA
+F https://www.ti.com/lit/ds/symlink/tps730.pdf
+$ENDCMP
+#
+$CMP TPS730285DBVT
+D Low Noise, High PSRR, 200mA Low Drop 2.85V Regulator, SOT-23-5
+K Fixed LDO 200mA
+F https://www.ti.com/lit/gpn/TPS730
```

Comment:
this should also get the direct .pdf link.

Reply:
Oh, my bad. Similar to the https://github.com/KiCad/kicad-symbols/pull/2515 pull request, I first created a standalone symbol and later realized that I could also create it as an alias of an already existing compatible symbol. I will delete this duplicate.